wave_matching/include/wave/matching/ground_segmentation.hpp

```diff
@@ -41,16 +42,14 @@ struct SignalPoint {
  */
 struct LinCell {
     EIGEN_MAKE_ALIGNED_OPERATOR_NEW
-    std::vector<PointXYZGD, Eigen::aligned_allocator<PointXYZGD>>
-      bin_points;  // all the points
-    std::vector<PointXYZGD, Eigen::aligned_allocator<PointXYZGD>>
-      obs_points;  // just the obs points
-    std::vector<PointXYZGD, Eigen::aligned_allocator<PointXYZGD>> drv_points;
-    std::vector<PointXYZGD, Eigen::aligned_allocator<PointXYZGD>>
-      ground_points;  // just the ground points
-    PointXYZGD prototype_point;
-    int cluster_assigned;  // what cluster is it assigned to
-    Vec3 obs_mean;         // mean of obstacle points
+    // Points in the input cloud are referred to by indices
+    std::vector<int> bin_indices;  // all the points
```

Comment:
Would it make more sensor for the type to be unsigned?

Reply:
PCL functions take `std::vector<int>`, that's why I used this